src/scripts/makeitsafe.js

```diff
@@ -0,0 +1,49 @@
+// TODO: Clean up. Not really wriotten for modules and a bit of a hack as is.
```

Comment:
Just a bit of a speil. I think to make this play better with modules, it should be rewritten